package.json

```diff
@@ -70,6 +70,7 @@
     "p-map": "5.5.0",
     "postinstall-postinstall": "2.1.0",
     "size-limit": "8.2.4",
+    "stylelint": "15.5.0",
```

Comment:
why does Preflight depend on Stylelint?

Reply:
types needed in Stylelint Check https://github.com/upleveled/preflight/blob/2a7b55f174b5ce88407d3d79c9c4fd2bc0f463b6/src/checks/stylelint.ts#L2